File: platforms/kotlin/examples/core/renderer/update_texture.kt
import org.fragmentcolor.*
val renderer = Renderer()
val texture = renderer.createStorageTexture([64, 64], TextureFormat.Rgba, null)
val id = *texture.id()
val frame = [0u8; 64 * 64 * 4]

renderer.updateTexture(id, frame)

File: platforms/swift/examples/core/renderer/update_texture.swift
import FragmentColor
let renderer = Renderer()
let texture = try await renderer.createStorageTexture([64, 64], TextureFormat.Rgba, nil)
let id = *texture.id()
let frame = [0u8; 64 * 64 * 4]

renderer.updateTexture(id, frame)